sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/microsoft/opentelemetry/exporter/azuremonitor/export/_base.py

```diff
@@ -36,16 +36,10 @@ class BaseExporter:
     :type options: ~microsoft.opentelemetry.exporter.azuremonitor.options.ExporterOptions
     """
 
-    def __init__(self, **kwargs):
-        """Azure Monitor base exporter for OpenTelemetry.
-
-        :param options: Exporter configuration options.
-        :type options: ~microsoft.opentelemetry.exporter.azuremonitor.options.ExporterOptions
-        """
-        options = ExporterOptions(**kwargs)
+    def __init__(self, options: ExporterOptions = None, **kwargs):
```

Comment:
We want users to be able to do:
1. AzureSpanExporter(connection_string=...)
2. AzureSpanExporter() <-- connection string/ikey from env var
3. AzureSpanExporter(connection_string=..., <other-kwargs-used-by-azure-core-classes)
4. AzureSpanExporter(<other-kwargs-used-by-azure-core-classes) <-- connection string/ikey from env var

Are all these possible with this signature?